packages/ee/billing/lib/constants.ts

```diff
@@ -1,23 +1,25 @@
 export enum ProductFeatureKeys {
```

Comment:
since this file contains constants, please use the upper case for all of them, you can take a look at the `COLOR_DEFAULTS` under `packages/lib/styling/constants.ts` for reference, its basically to be consistent around constants, like you've done with LIMITS 🙏